Sources/AltSwiftUI/Source/Views/Controls/SecureField.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  SecureField.swift
+//  AltSwiftUI
+//
+//  Created by Elvis Lin on 2020/12/1.
+//
+
+import UIKit
+
+/// A view into which the user securely enters private text.
+public struct SecureField: View {
+    public var viewStore = ViewValues()
+
+    let title: String
+    let onCommit: () -> Void
+    var text: Binding<String>
+    var isFirstResponder: Binding<Bool>?
+
+    public var body: View {
+        TextField(title, text: text, onCommit: onCommit)
```

Comment:
1. You need to set secure entry to true in the initializer
2. You need to pass the value of `isFirstResponder` to this view through a modifier

Reply:
Thanks for your suggestion. I only keep a TextField inside SecureField now.